### DISCUSSION TITLE
Is it possible to use H3 with flutter?

### DISCUSSION THREAD

**Discussion by YousefSkandari** [uber/h3#630] [Q&A]
I've been having some issues integrating h3 into a flutter based mobile app. 
I'm using mapbox for the map and can't seem to get the hexagons to be viewable. 
Am I completely missing something or is it possible to do?

**Comment by nrabinowitz** [comment]
While I have never used them, you might be interested in the Dart/Flutter bindings here: https://github.com/festelo/h3_dart

Beyond that, I'd suggest this is a better question for StackOverflow, as it is likely more specific to Flutter than H3.
